cmd/tracking.go

```
@@ -137,15 +140,30 @@
 				log.Println("forced poll")
-				ch.gameTracker.Poll(ctx, cancel, session, onNewMatch)
+				match, err := ch.gameTracker.Poll(ctx, session)
+				if err != nil {
+					cancel()
+					return
+				}
+				onNewMatch(match)
 			case <-ticker.C:
 				log.Println("polling")
-				ch.gameTracker.Poll(ctx, cancel, session, onNewMatch)
+				ch.gameTracker.Poll(ctx, session)
```

golangci-lint (1.64.8)

Error return value of `ch.gameTracker.Poll` is not checked

(errcheck)